python/flarm/packetize.py

```diff
@@ -116,16 +116,40 @@ class packetize(gr.basic_block):
     def __init__(self, num_channels, first_channel, rxid, reflat, reflon):
         gr.basic_block.__init__(self,
             name="packetize",
-            in_sig=[numpy.int8]*num_channels,
-            out_sig=[numpy.int8])
+            in_sig=[numpy.int8,]*num_channels,
+            out_sig=[numpy.int8,])
         self.first_channel = first_channel
         self.rxid = rxid
         self.reflat = int(reflat * 1e7)
         self.reflon = int(reflon * 1e7)
 
-    def forecast(self, noutput_items, ninput_items_required):
-        for channel in range(len(ninput_items_required)):
-            ninput_items_required[channel] = 5000
+    def forecast(self, noutput_items, ninputs):
+        # ninputs is the number of input connections
+        # setup size of input_items[i] for work call
+        # the required number of input items is returned
+        #   in a list where each element represents the
+        #   number of required items for each input
+        ninput_items_required = [noutput_items] * ninputs
+        for i in range(ninputs):
+            ninput_items_required[i] = 5000
+        return ninput_items_required
```

Comment:
This could be simplified to a one-liner:
```suggestion
        return [5000] * ninputs
```